web-client/public/js/graph.js

```diff
@@ -1484,39 +1484,42 @@ export var drawGraph = function (network, nodeColoring) {
     modifyChargeParameter(-50);
     modifyLinkDistanceParameter(500);
 
-// figure out this code???
-    var changeSliderValue = function (slider, item) {
-        var value = slider === "link" ? linkDistValidator($(item).val()) :
-            chargeValidator($(item).val());
+    const changeSliderValue = function (slider) {
         if (slider === "link") {
-            grnState.linkDistanceSlider.currentVal = value;
-            modifyLinkDistanceParameter(value);
-            $(LINK_DIST_VALUE).text(value);
-            $(LINK_DIST_SLIDER_ID).val(value);
-            $(LINK_DIST_MENU).val(value);
+            modifyLinkDistanceParameter(grnState.linkDistanceSlider.currentVal);
```

Comment:
@dondi I have implemented somewhat of an MVC within graph.js, however I am hesitant to move it, because it would require me to call changeSliderValue (ideally contained within update-app.js) directly from within setup-handlers.js. A workaround would be for a T/F value to be set in the handler function, however that would have to be reset from within update-app.

Reply:
Wait, do I not actually need to force changeSliderValue to be called?